usvm-go/src/main/kotlin/org/usvm/machine/GoTypeSystem.kt

```diff
@@ -0,0 +1,40 @@
+package org.usvm.machine
+
+import org.usvm.bridge.GoBridge
+import org.usvm.types.USupportTypeStream
+import org.usvm.types.UTypeStream
+import org.usvm.types.UTypeSystem
+import kotlin.time.Duration
+
+class GoTypeSystem(
+    private val bridge: GoBridge,
+    override val typeOperationsTimeout: Duration
+) : UTypeSystem<GoType> {
+    private val anyType = bridge.getAnyType()
```

Comment:
Probably, it is better to rename it to avoid collision with any type of the system